Controller/Checkout/Cart.php

```diff
@@ -32,6 +32,8 @@ public function __construct(
     public function execute()
     {
         $quoteId = $this->request->getParam('quote_id');
```

Comment:
we shouldn't need to call the request getParam and getParams, this can be cleaned up.

Reply:
do you mean call $this->request->getParams() once and then grab quoteId and then unset quoteId?

Reply:
yea that would be great